yt/frontends/enzo/data_structures.py

```diff
@@ -781,7 +781,7 @@ def _parse_enzo3_parameter_file(self, f):
 
     def _parse_enzo2_parameter_file(self, f):
         for line in (l.strip() for l in f):
-            if len(line) < 2: continue
+            if (len(line) < 2) or ("=" not in line): continue
```

Comment:
It's fine to leave as is, but just as an fyi, the parentheses are not strictly necessary.